rllib/agents/ppo/ppo_tf_policy.py

```diff
@@ -188,13 +190,16 @@ def postprocess_ppo_gae(
         last_r = 0.0
     # Trajectory has been truncated -> last r=VF estimate of last obs.
     else:
-        next_state = []
-        for i in range(policy.num_state_tensors()):
-            next_state.append(sample_batch["state_out_{}".format(i)][-1])
-        last_r = policy._value(sample_batch[SampleBatch.NEXT_OBS][-1],
-                               sample_batch[SampleBatch.ACTIONS][-1],
-                               sample_batch[SampleBatch.REWARDS][-1],
-                               *next_state)
+        if policy.config["_use_trajectory_view_api"]:
```

Comment:
Eventually remove boilerplate input-dict generation for non-traj. view case (in else block).